packages/now-cli/test/integration.js

```diff
@@ -2279,17 +2279,19 @@ test('should prefill "project name" prompt with now.json `name`', async t => {
 
   let isDeprecated = false;
 
-  await waitForPrompt(now, chunk => {
-    if (chunk.includes('The `name` property in now.json is deprecated')) {
+  now.stderr.on('data', data => {
+    if (
+      data.toString().includes('The `name` property in now.json is deprecated')
+    ) {
       isDeprecated = true;
     }
+  });
 
+  await waitForPrompt(now, chunk => {
```

Comment:
Are those changes necessary?

Reply:
Yes, because we can't tell the user that `.name` is deprecated before we know where `now.json` is